VideoLocker/res/values/strings.xml

```diff
@@ -477,6 +477,14 @@
     <!-- In Sign up screen -->
     <string name="or_sign_up_with_email">or sign up with email</string>
 
+    <!-- New Logistration Launch screen text -->
+    <!-- Button leading user to create account screen -->
+    <string name="launch_register">Register</string>
+    <!-- Button leading user to Find Courses screen from the launch screen -->
+    <string name="launch_discover_courses">@string/find_courses_title</string>
```

Comment:
@catong Yes, that's ok for now. I'm also not convinced as to why the word Discover Courses is used here but not everywhere. We use: Find, Explore, Discovery & Search, depending on where you look. :) 

We can review and replace text wherever we need to, but I don't want to hold this work up on the basis of that consistency for now. Once someone is signed in they won't generally see this button again. Thanks for checking!